dll/win32/shell32/shellmenu/CMenuToolbars.cpp

```diff
@@ -106,7 +106,7 @@ HRESULT CMenuToolbarBase::OnWinEvent(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM
             break;
 
             // Unknown
-        case -714: return S_FALSE;
+        case (UINT)-714: return S_FALSE;
```

Comment:
This is weird.
@yagoulas Do you know if there are better id's for this or so?

Reply:
This looks like TBN_DRAGOUT 
#define TBN_FIRST (0U-700U)
#define TBN_LAST (0U-720U)
#define TBN_DRAGOUT (TBN_FIRST - 14)